R/import.R

```diff
@@ -4,21 +4,42 @@
   good_path <- .doc_path(path)
 
   if (fs::file_exists(fs::path_abs("README.md", start = path))) {
-    fs::file_copy(
-      fs::path_abs("README.md", start = path),
-      paste0(good_path, "/README.md"),
-      overwrite = TRUE
-    )
+
+    if (.doc_type(path) == "quarto_website") {
```

Comment:
should it simply be "quarto"? there's not much confusion possible with another format